content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -47,8 +47,8 @@ Analyze the current live cluster, simulating the effect of applying additional y
 
 {{< text syntax=bash snip_id=analyze_sample_destrule >}}
 $ istioctl analyze @samples/bookinfo/networking/bookinfo-gateway.yaml@ @samples/bookinfo/networking/destination-rule-all.yaml@
-Error [IST0101] (Gateway default/bookinfo-gateway samples/bookinfo/networking/bookinfo-gateway.yaml:7) Referenced selector not found: "istio=ingressgateway"
-Error [IST0101] (VirtualService default/bookinfo samples/bookinfo/networking/bookinfo-gateway.yaml:39) Referenced host not found: "productpage"
+Error [IST0101] (Gateway default/bookinfo-gateway samples/bookinfo/networking/bookinfo-gateway.yaml:9) Referenced selector not found: "istio=ingressgateway"
```

Comment:
I do know @frankbu mentioned we may be able to make this better by not looking for specific line numbers. I haven't seen a line number failure here before as I suspect the samples are pretty static.